ddl/syncer.go

```diff
@@ -83,13 +88,23 @@ type SchemaSyncer interface {
 	// the latest schema version. If the result is false, wait for a while and check again util the processing time reach 2 * lease.
 	// It returns until all servers' versions are equal to the latest version or the ctx is done.
 	OwnerCheckAllVersions(ctx context.Context, latestVer int64) error
+
+	// GetServerInfoFromPD get the DDL_id server information from PD.
+	GetServerInfoFromPD(ctx context.Context, id string) (*util.DDLServerInfo, error)
+	// GetAllServerInfoFromPD get all DDL servers information from PD.
+	GetAllServerInfoFromPD(ctx context.Context) (map[string]*util.DDLServerInfo, error)
+	// StoreSelfServerInfoToPD store DDL server information to PD.
+	StoreSelfServerInfoToPD(ctx context.Context, info *util.DDLServerInfo) error
+	// RemoveSelfServerInfoFromPD remove DDL server information from PD.
+	RemoveSelfServerInfoFromPD() error
```

Comment:
I think we don't need "ToPD" or "FromPD". The functions in this file are all used to interact with PD.